conda_forge_tick/lazy_json_backends.py

```diff
@@ -308,6 +308,18 @@ def hgetall(self, name: str, hashval: bool = False) -> Dict[str, str]:
         )
 
 
+def _test_and_raise_besides_file_not_exists(e):
+    if isinstance(e, github.UnknownObjectException):
+        pass
+    elif isinstance(e, github.GithubException):
+        if e.status == 404 and "No object found" in e.data["message"]:
+            pass
+        else:
+            raise e
+    else:
+        raise e
```

Comment:
```suggestion
def _test_and_raise_besides_file_not_exists(e):
    if isinstance(e, github.UnknownObjectException):
        return
    if isinstance(e, github.GithubException) and e.status == 404 and "No object found" in e.data["message"]:
        return
    raise e
```
You can reduce the complexity here.